roles/devscripts/tasks/135_patch_src.yml

```diff
@@ -49,14 +31,18 @@
     _vm_setup_vars: "{{ vm_setup_data.content | b64decode | from_yaml }}"
 
 # Note: Override When external network is being reused.
-- name: Verify dev-scripts vm_setup_vars to reflects external network address
+- name: Ensure dev-scripts vm_setup_vars reflects external network address
```

Comment:
You can perfectly squash this task with the next one resulting in a simpler approach, without the useless `_modify_vm_setup` variable and the need to run two tasks (even three, cause the previous set_fact is useless too)

Reply:
Thanks.
Yes, that make sense.